rskj-core/src/main/resources/reference.conf

```diff
@@ -296,6 +296,8 @@ sync {
     # block chain synchronization can be: [true/false]
     enabled = true
 
+    mode = full
```

Comment:
not sure we need a new config. We could simply make use of existing one -  `sync.snapshot.client.enabled`

Reply:
Agreed.
But if we still proceed having a new config, we should at least have a comment explaining the use.

Reply:
solved, removed the new config